include/dfu/dfu_target_stream.h

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright (c) 2020 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+/** @file dfu_target_stream.h
+ *
+ * @defgroup dfu_target_stream Flash stream DFU Target
+ * @{
+ * @brief Provides an API for other DFU targets that need to write a large
+ *        object to flash.
+ */
+
+#ifndef DFU_TARGET_STREAM_H__
+#define DFU_TARGET_STREAM_H__
+
+#include <stddef.h>
+#include <storage/stream_flash.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct stream_flash_ctx *dfu_target_stream_get_stream(void);
+
+/**
+ * @brief Initialize dfu target.
+ *
+ * @param[in] id The identifier of the stream, used for storing settings.
```

Comment:
Minor: Suggested to tab-indent the descriptions from the arg name to improve readability.

Reply:
Fixed by init struct.